woocommerce/woocommerce-gutenberg-products-block
# Add custom className support to Cart Inner Blocks

Adds classname support to all cart inner blocks, based on #4978

Fixes #4962


### Testing

How to test the changes in this Pull Request:

1. Smoke test cart
2. Confirm you can add classnames to cart inner blocks and then are shown on the frontend
   - Select inner block e.g. proceed to checkout block.
   - Toggle open the advanced section in the inspector
   - Add custom CSS class
   - View frontend
   - Inspect the element using browser tools and confirm the class is present
